babel.config.js

```diff
@@ -29,20 +29,28 @@ module.exports = {
         'transform-react-remove-prop-types',
         '@babel/plugin-transform-react-inline-elements',
         '@babel/plugin-transform-react-constant-elements',
-        ['import', { libraryName: 'antd', style: true }, 'import-antd']
+        ['import', { libraryName: 'antd', style: true }, 'import-antd'],
+        ['import', { libraryName: 'lodash' }, 'import-lodash']
```

Comment:
@mac-wednesday shall we add for @antd/design-icons as well ?

Reply:
i don't think it imports the icons there's some change in the webpack config for it